### ISSUE TITLE
Placeholder for attachments

### ISSUE THREAD

**Issue by mStirner** [keepassxreboot/keepassxc#6947]
## Summary
I found out about "placeholders": https://keepass.info/help/base/placeholders.html
Is there a way to use "Attachements" placeholders? Found this here https://sourceforge.net/p/keepass/discussion/329220/thread/4b2ebfb6/#6a37 but that does not work for me (since its a feature request too)

Debug-Information
```
KeePassXC - Version 2.3.1
Revision: 2fcaeea

Bibliotheken:
- Qt 5.9.5
- libgcrypt 1.8.1

Betriebssystem: Ubuntu 18.04.5 LTS
CPU-Architektur: x86_64
Kernel: linux 5.4.0-86-generic

Aktivierte Erweiterungen:
- Auto-Type
- Browser Integration
- Legacy Browser Integration (KeePassHTTP)
- SSH Agent
- YubiKey
```

## Examples
Use of `cmd://{ATTACHMENT:1}` in url field to execute shell/bash scripts stored as attachment.

## Context
Would be great to copy other attachments somewhere on the system and/or transfer them over ssh to another machine.

**Comment by droidmonkey** [comment]
Why? Just enter the information into a custom attribute, or just write the command out directly. There is no benefit to burying this in an attachment.

**Comment by mStirner** [comment]
Well, thanks for the tipp.
But how do i get the content of a attachment? 

The script thing was just a (bad) example.

I would really think we should discuses this more then just close this issue without talking about it.

**Comment by droidmonkey** [comment]
An attachment is a file, it is meant to be opened and/or saved by a user. If you want content to use as placeholders or something else then put that in a custom attribute.

**Comment by mStirner** [comment]
I understand that, and you have a valid point. But why dont you want to improve the software to be able get the content of attachments as placeholders? Its a usefully feature.

Now i have to store it as a attachment & custom attribute, which is unnecessary overhead.
Furthermore, custom attributes a not good for binary files.

How should i handle  binary files in custom attributes?

---
EDIT: Perhaps this is a xy problem from my side.
Is there a way to get the content of a attachment from bash/cli

**Comment by droidmonkey** [comment]
Accessing attachments from cli is about to be merged to develop branch and will be part of 2.7.0. You can see a preview of that here: #5538
